### DISCUSSION TITLE
Setting the sensor parameter remotely (http request)

### DISCUSSION THREAD

**Discussion by juanandrescl** [hafidhh/FlowSensor-Arduino#18] [Q&A]
Hi All, 
Please I need some help here.
Accoording to the examples,  the count() function  and the sensor type must be declared/defined  outside the setup() function.

Since I can not define functions inside de the setup() functions..

**what would be a good approach for retrieving the "type" variable from an HTTP request?**

Thanks in advance

#define type YFS201
#define pin D2 // pin -> interrupt pin
```

FlowSensor Sensor(type, pin);

void count()
{
	Sensor.count();
}

void setup() {
	Serial.begin(115200);
	Sensor.begin(count);
}
```

**Comment by hafidhh** [comment]
Hi, 
sorry for taking so long to respond
at newest version i have create function to set type and pin. you can use it by Sensor.setType(type) and Sensor.setPin(pin).
Thanks
